llvm/test/Other/dump-with-filter.ll

```diff
@@ -0,0 +1,14 @@
+;; Make sure we can run -filter-print-funcs with -ir-dump-directory.
+; RUN: rm -rf %t/logs
```

Comment:
do we need to `mkdir %t/logs`?

Reply:
It's automatically generated by `-ir-dump-directory`, so I don't think so.